src/main/kotlin/net/irisshaders/lilybot/utils/DatabaseHelper.kt

```diff
@@ -148,20 +150,82 @@ object DatabaseHelper {
 	/**
 	 * Add the given [newStatus] to the database
 	 *
-	 * @param [newStatus] The new status you wish to set
+	 * @param newStatus The new status you wish to set
 	 * @author NoComment1105
 	 */
 	suspend fun setStatus(newStatus: String) {
 		val collection = database.getCollection<StatusData>()
 		collection.deleteOne(StatusData::key eq "LilyStatus")
 		collection.insertOne(StatusData("LilyStatus", newStatus))
 	}
+
+	/**
+	 * Gets the given tag using it's [name] and returns the [column] selected. If the tag does not exist
+	 * it will return null
+	 *
+	 * @param guildId The ID of the guild the command was run in
+	 * @param name The named identifier of the tag
+	 * @param column The tag data you are requesting
+	 * @return The requested [column] value or null
+	 * @author NoComment1105
+	 */
+	suspend fun getTag(guildId: Snowflake, name: String, column: String): Any? {
```

Comment:
I would rename these inputs to be prefixed with input. Same for the other new functions.